core/io/json.cpp

```diff
@@ -211,23 +211,20 @@ Error JSON::_get_token(const CharType *p_str, int &index, int p_len, Token &r_to
 										r_err_str = "Unterminated String";
 										return ERR_PARSE_ERROR;
 									}
-									if (!((c >= '0' && c <= '9') || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F'))) {
+									if (!((c >= '0' && c <= '9') || (c >= 'A' && c <= 'F') || (c >= 'a' && c <= 'f'))) {
```

Comment:
What's the point of this change?

Reply:
In ascii big letters are before small, so code can below can look
```
	if (c <= '9') {
		v = c - '0';
	} else if (c <= 'F') {
		v = c - 'A';
		v += 10;
	} else {
		v = c - 'a';
		v += 10;
	}
```
instead something like
```
	if (c <= '9') {
		v = c - '0';
	} else if (c >= 'a' && c <= 'f') {
		v = c - 'a';
		v += 10;
	} else if (c >= 'A') {
		v = c - 'a';
		v += 10;
	}
```